app/scripts/angular-apimock.js

```diff
@@ -45,13 +45,49 @@ angular.module('apiMock', [])
 		var config = {
 			mockDataPath: '/mock_data',
 			apiPath: '/api',
-			disable: false
+			disable: false,
+			stripQueries: true
 		};
 		var fallbacks = [];
 
 		// Helper methods
 		//
 
+		function serialize(obj) {
+			var str = [];
+
+			obj = sortObjPropertiesAlpha(obj);
+			for(var p in obj){
+				if (obj.hasOwnProperty(p)) {
+					var value = encodeURIComponent(obj[p]);
+					//If the value is a string make it lowercase
+					if (typeof value === 'string') {
```

Comment:
Is there a case where `encodeURIComponent` returns a non-string type? I can't see a mention of it on [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent). But if there is, let's prove it with a unit test.